app/bundles/AddonBundle/Integration/AbstractIntegration.php

```diff
@@ -856,6 +895,41 @@ public function getAuthenticationUrl ()
     }
 
     /**
+     * Get request token for oauth1a authorization request
+     *
+     * @param array $settings
+     *
+     * @return mixed|string
+     */
+    public function getRequestToken($settings = array())
+    {
+        // Child classes can easily pass in custom settings this way
+        $settings = array_merge(array('authorize_session' => true, 'append_callback' => false, 'ssl_verifypeer' => false), $settings);
+
+        $url = $this->getRequestTokenUrl();
+        if (!empty($url)) {
+            $result = $this->makeRequest(
+                $url,
+                array(),
+                'POST',
+                $settings
+            );
+        }
+
+        return $result;
```

Comment:
My bad; `return $result` should probably go inside the `if (!empty($url)) {}` or add `$result = ''; before the if statement. $url shouldn't ever be empty since the same integration should be handling both functions but, just in case, should prevent a PHP notice.